gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/test/java/io/gravitee/rest/api/management/rest/spring/ResourceContextConfiguration.java

```diff
@@ -509,4 +510,9 @@ public VerifyApiPathDomainService verifyApiPathDomainService(
     public CloseSubscriptionDomainService closeSubscriptionDomainService() {
         return mock(CloseSubscriptionDomainService.class);
     }
+
+    @Bean
```

Comment:
Is it necessary to mock the ParametersQueryService? The InMemory service is used in the Resource test, so I would imagine you can delete this bean.

Reply:
Yes it is unfortunately. The ManagementUIResourceTest use the inmemory bean so here it's ok. Howerver, for the other ResourceTest we need to have a bean to inject to avoid to have an initialisation error saying that the use case can ne instantiate because no ParameterQueryService bean in available. This is something we already have on other service like this one.